doc/source/release-notes/Percona-Server-5.7.30-33.rst

```diff
@@ -0,0 +1,61 @@
+.. _PS-5.7.30-33:
+
+================================================================================
+*Percona Server for MySQL* 5.7.30-33
+================================================================================
+
+:Date: May 25, 2020
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.30-33
+includes all the features and bug fixes available in
+`MySQL 5.7.30 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-30.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+New Features
+================================================================================
+
+* :jirabug:`PS-6902`: Documented the new variable enable_insert_with_update_caching.
+* :jirabug:`PS-4464`: Exposed the last global transaction identifier (GTID) executed for a CONSISTENT SNAPSHOT.
+* :jirabug:`PS-6886`: Documented a new variable rocksdb_cache_dump which includes RocksDB block cache content in a core dump.
+* :jirabug:`PS-6890`: Documented the new variable blind_delete_primary_key
+* :jirabug:`PS-6885`: Documented the new variable rocksdb_rollback_on_timeout which allows the rollback of an entire transaction on timeout.
+* :jirabug:`PS-6910`: Documented the new variable rocksdb_stats_level.
+* :jirabug:`PS-6901`: Documented the new variable read_free_rpl
+* :jirabug:`PS-6891`: Documented the new variable master_skip_tx_api
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-6984`: Updated the zstd submodule.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-6979`: Modified processing to call clean up functions as part of event (Upstream :mysqlbug:`99200`)
+* :jirabug:`PS-6860`: The innodb_buffer_pool_pages_LRU_flushed variable did not work.
+* :jirabug:`PS-6811`: Service failed to start asserting ACL_PROXY_USER::check_validity if --skip-name-resolve=1 and there is a Proxy user. (Upstream :mysqlbug:`98908`)
+* :jirabug:`PS-6112`: Inconsistent Binlog_snapshot_gtid when mysqldump was used with --single-transaction.
+* :jirabug:`PS-6945`: The tokubackup was broken since it no longer exported the correct large file API.
+* :jirabug:`PS-6856`: corrupted binlogs in PS 5.7.28 and 5.7.29 (Upstream :mysqlbug:`97531`)
+* :jirabug:`PS-6946`: Sanitizers found memory use after free and other issues in tokubackup.
+* :jirabug:`PS-5893`: Added support for running multiple instances with systemD on Debian.
+* :jirabug:`PS-4573`: The 5.7 patch created a mysqld.cnf file which caused other configurations from functioning correctly.
+* :jirabug:`PS-7020`: Converted MTR tests from python2 to python3.
```

Comment:
In Ubuntu 20.04, only python3-mysqldb is available. We converted the MTR tests to support both Python2 and Python3. The scripts require Python 2.6 or newer.